elm-package.json

```diff
@@ -1,15 +1,19 @@
 {
     "version": "1.0.0",
-    "summary": "Game of pong, based on Evan's blog post. Enhanced with some new features",
-    "repository": "https://github.com/davydog187/elm-pong.git",
+    "summary": "helpful summary of your project, less than 80 characters",
+    "repository": "https://github.com/user/project.git",
```

Comment:
Whoops.  I fixed it.  Was generating from scratch with each new elm version and forgot that there was hand written information in there.